tests/integration/any_charm.py

```diff
@@ -13,21 +13,43 @@
 
 
 class AnyCharm(AnyCharmBase):
-    """Execute a simple web-server charm to test the ingress-proxy relation"""
+    """Execute a simple web-server charm to test the ingress-proxy relation.
+
+    Attrs:
+        ingress: The attribute that mimics a real ingress relation.
+    """
 
     def __init__(self, *args, **kwargs):
+        """Init function for the class.
+
+        Args:
+            args: list of arguments
+            kwargs: list of keyword arguments
+        """
         super().__init__(*args, **kwargs)
         self.ingress = IngressRequires(
             self,
             {"service-hostname": self.app.name, "service-name": self.app.name, "service-port": 80},
         )
 
     def update_ingress(self, ingress_config):
+        """Update Ingress config.
+
+        Args:
+            ingress_config: New Ingress configuration to be applied.
+        """
         self.ingress.update_config(ingress_config)
 
     @staticmethod
     def start_server(port: int = 80):
-        """Start an HTTP server daemon."""
+        """Start an HTTP server daemon.
+
+        Args:
+            port: The port where the server is connected, 80 by default.
```

Comment:
Do you thin it is worth including the default value in the description given it is already called out in the signature?